target/linux/ipq806x/files-4.19/arch/arm/boot/dts/qcom-ipq8065-xr500.dts

```diff
@@ -0,0 +1,531 @@
+#include "qcom-ipq8065.dtsi"
```

Comment:
Please add an SPDX license identifier.

Reply:
As we are firmly with kernel 5.4 for ipq806x, I see no actual need to create dts also for the deprecated 4.19.
(but it doesn't actually hurt, so maybe it can stay although there is no real need for that)

Reply:
@hnyman is right. The only downside of having no DTS in 4.19 would be that the build fails if somebody just wanted to build all devices (without looking) for 4.19 for some testing purposes. But that will be the case anyway sooner or later (as other people will do 5.4-only ...)

BTW my remarks on 4.19-DTS obviously should also be applied to 5.4-DTS, I just didn't want to do it twice ...